src/account/PluginManager2.sol

```diff
@@ -80,22 +73,23 @@ abstract contract PluginManager2 {
             }
         }
 
-        if (isGlobal) {
-            if (_storage.validationData[validationFunction].isGlobal) {
-                revert GlobalValidationAlreadySet(validationFunction);
-            }
-            _storage.validationData[validationFunction].isGlobal = true;
+        if (validationConfig.isGlobal()) {
+            _validationData.isGlobal = true;
+        }
+
+        if (validationConfig.isSignatureValidation()) {
+            _validationData.isSignatureValidation = true;
```

Comment:
If they are false, why dont we want to update to false (if current value is true)? It won't cost extra if it is updating false=> false or true=>true.

Reply:
I did not think of how this could be used as an edit feature, but yes it looks totally possible if we remove these `if` checks. Done.